### ISSUE TITLE
Find a way to gather download counts information

### ISSUE THREAD

**Issue by jcs090218** [jcs-emacs/elpa#2]
It would be great if we could have a `download-counts.json` file like MELPA.

**Comment by SamB** [comment]
You'd need access to server logs for that, and I'm fairly certain github pages does not support that.

**Comment by jcs090218** [comment]
Yeah, I was hoping there was another workaround, but it seems impossible for now. :(

Thanks for double-confirming this!
